completions/ssh

```diff
@@ -5,7 +5,7 @@ _ssh_queries()
     local queries=$(_comp_xfunc_ssh_query "$1" help)
     [[ $queries ]] || queries="cipher cipher-auth mac kex key key-cert
          key-plain key-sig protocol-version compression sig ciphers macs
-         kexalgorithms pubkeyacceptedkeytypes hostkeyalgorithms
+         kexalgorithms pubkeyacceptedalgorithms hostkeyalgorithms
```

Comment:
This is the fallback hardcoded set of query types in case `ssh -Q help` doesn't give us a set of supported ones to work with. I don't think we need to cater for the change in question here.